sources/execute.c

```diff
@@ -1851,12 +1851,13 @@ int GetFirstBracket(WORD *term, int num)
  		#[ GetFirstTerm :
 */
 
-int GetFirstTerm(WORD *term, int num)
+int GetFirstTerm(WORD *term, int num, int pre)
 {
 /*
 		Gets the first term of the expression 'num'
 		Puts it in term.
 		Routine should be thread-safe
+		The "pre" arg denotes a pre-processor call (1) or not (0).
```

Comment:
How about reformatting the comments for this function into Doxygen style now that you've added the `pre` parameter and its explanation?

```c
/*
 		#] GetFirstBracket : 
 		#[ GetFirstTerm :
*/

/**
 * Gets the first term...
 *
 * @param[in] term ...
 * @param[in] num ...
 * @param[in] pre ...
 * @return ...
 */
int GetFirstTerm(WORD *term, int num, int pre)
```

Reply:
Sure, done.